packages/synapse-react-client/src/lib/containers/auth/LoginPage.tsx

```diff
@@ -14,6 +15,14 @@ import useLogin from '../../utils/hooks/useLogin'
 import LoginForm from './LoginForm'
 import { StyledComponent } from '@emotion/styled'
 
+const LOST_ACCOUNT_SERVICE_DESK_ISSUE_TITLE = encodeURIComponent(
+  'Lost access to my Synapse account',
+)
+const LOST_ACCOUNT_SERVICE_DESK_DESCRIPTION = encodeURIComponent(
+  '<Please provide your username and/or email address associated with your account.>',
+)
```

Comment:
How could we make this better?

Reply:
Is there a way to disable 2FA for a user once their identity is verified?  I see https://rest-docs.synapse.org/rest/DELETE/2fa.html, but that's for the current user only.  Would this need to be a direct database update?
@marcomarasca